torch/lib/TH/generic/THTensorMath.c

```diff
@@ -642,6 +642,16 @@ void THTensor_(sub)(THTensor *r_, THTensor *t, real value)
   THTensor_(add)(r_, t, -value);
 }
 
+void THTensor_(add_scaled)(THTensor *r_, THTensor *t, real value, real alpha)
+{
+  THTensor_(add)(r_, t, value * alpha);
```

Comment:
Scaled version of add does
```
r_ = alpha * (t + value)
```
not
```
r_ = t + (value * alpha)
```
Are you sure this is correct?

Reply:
This does:

```
r_ = t + (value * alpha)
```

Which is correct and matches the behavior of `cadd`